src/ImageManipulation.php

```diff
@@ -1057,4 +1062,98 @@ protected function castDimension($value, $dimension)
         }
         return $value;
     }
+
+    private function copyImageBackend()
```

Comment:
This bit was mostly extracted out of `manipulate`. I tried calling `manipulate` directly but it just does too many thing to work well.

Reply:
does it still work calling `manipulate()` though? If so I'd be inclined to still use it.  Just looking at a comment within `mainpulate()`

```
        // Pass the manipulated image backend down to the resampled image - this allows chained manipulations
        // without having to re-load the image resource from the manipulated file written to disk
```

Possible but not calling manipulate we'll create weird edge case bugs?  e.g. setWidth(300).setLazyLoad(false) vs setLazyLoad(false).setWidth(300) will behave differently somehow?  I just don't know

Reply:
`manipulate` will only call the callback if the variant does not exist. Since we're not actually creating a variant. This means there's a good chance our callback will never be called.

I'm pretty sure `setWidth(300).setLazyLoad(false)` and `setLazyLoad(false).setWidth(300)` will end up in the same place, but I'll double check.